### ISSUE TITLE
Support for Base URLs using Environment Variables

### ISSUE THREAD

**Issue by chadcrowell** [vaersaagod/geomate#14]
The sites in this Craft install all get there Base URLs from ENV vars:
[https://d.pr/i/dlmcNP](https://d.pr/i/dlmcNP)

When I am testing the redirectMap locally with these config settings:

```
return [
    'cacheEnabled' => true,
    //'dbPath' => '',
    'autoRedirectEnabled' => true,
    'redirectMap' => [
    	'www' => 'us'
    	],
    'redirectIgnoreBots' => false,
    //use the following for testing locally
    'redirectIgnoreAdmins' => false,
    'forceIp' => '72.35.132.254'
 ];
```

The redirect happens, but the resulting URL is `http://store.gcx.craft/$SITE_URL_WWW` - it appears that possibly GeoMate isn't parsing the ENV Vars in the site settings. If that's not the case and I have a config issue, let me know what you think may fix it. Otherwise, parsing ENV Vars is a feature request.

Thanks.

**Comment by chadcrowell** [comment]
I did end up setting up config file aliases for the ENV Vars and using the alias in the site URL setting does allow Geomate to redirect appropriately. I would like to see support for ENV vars directly though. Thanks.

**Comment by aelvan** [comment]
Thanks for reporting, this has been fixed in v1.0.6.
